releases/1.1.1-SNAPSHOT.yml

```diff
@@ -0,0 +1,3 @@
+version: 1.1.1-SNAPSHOT
+rpm_version: 1.1.1~SNAPSHOT
+deb_version: 1.1.1~SNAPSHOT
```

Comment:
Yes, it's because for apt and you: 1.1.1-SNAPSHOT > 1.1.1 but 1.1.1~SNAPSHOT < 1.1.1.

We want 1.1.1 to be installable over the snapshot, hence the ~.